src/iadmin.cpp

```diff
@@ -2585,6 +2615,24 @@ usage( char *subOpt ) {
         ""
     };
 
+    char* delete_session_tokens_usage[] = {
+        "delete_session_tokens expired|all [user[#zone]]",
+        " ",
+        "Delete session tokens from the catalog.",
+        " ",
+        "If 'expired' is specified, only expired session tokens will be removed.",
+        "If 'all' is specified, all session tokens will be removed.",
+        " ",
+        "If a user is specified, only session tokens belonging to that user will be removed.",
+        "The user's zone can optionally be specified in '#'-delimited format.",
+        " ",
+        "Examples:",
+        "iadmin delete_session_tokens all (removes all session tokens)",
+        "iadmin delete_session_tokens expired (removes all expired session tokens)",
+        "iadmin delete_session_tokens all alice (removes all session tokens for user alice)",
+        "iadmin delete_session_tokens expired alice (removes all expired session tokens for user alice)",
```

Comment:
I'm wondering if the descriptions should be on their own line, to guard against readers misinterpreting how to use these.

Consider looking at how various linux commands present examples.

Reply:
How does this look?
```suggestion
        "To remove all session tokens:",
        " $ iadmin delete_session_tokens all",
        "To remove all expired session tokens:",
        " $ iadmin delete_session_tokens expired",
        "To remove all session tokens for user alice:"
        " $ iadmin delete_session_tokens all alice",
        "To remove all expired session tokens for user alice:",
        " $ iadmin delete_session_tokens expired alice",
```

Reply:
That works.